ffi/capi/src/lib.rs

```diff
@@ -17,3 +21,7 @@ pub mod provider;
 
 #[cfg(target_arch = "wasm32")]
 mod wasm_glue;
+
+// Assume "none" is FreeRTOS for now
```

Comment:
Nit: Issue to track how to relax this assumption

Reply:
https://github.com/unicode-org/icu4x/issues/891

Reply:
Nit: Please add a reference to #891 from the code to make it more discoverable